packages/core/lib/copy.js

```diff
@@ -12,51 +12,53 @@ var cpr_options = {
 // won't override individual files. If a file exists, it will
 // simply move onto the next file.
 
-var copy = function(from, to, extra_options, callback) {
-  if (typeof extra_options === "function") {
-    callback = extra_options;
-    extra_options = {};
-  }
+const copy = function(from, to, extra_options) {
+  const options = _.merge(_.clone(cpr_options), extra_options);
 
-  var options = _.merge(_.clone(cpr_options), extra_options);
-
-  cpr(from, to, options, function(err, files) {
-    var new_files = [];
+  return new Promise((resolve, reject) => {
+    cpr(from, to, options, function(err, files) {
+      if (err) {
+        reject(err);
```

Comment:
This needs a `return;` after the `reject(err);` so the `resolve` isn't called as well.

Reply:
Good catch!